ntbs-service/Pages/Notifications/Edit/Children/NewManualTestResult.cshtml

```diff
@@ -0,0 +1,8 @@
+@page "/Notifications/{NotificationId:int}/Edit/ManualTestResult/{handler?}"
```

Comment:
Since we've arrived at splitting these up anyways, do we want to have `/Edit/ManualTestResult/New` as the page url here?

Reply:
Ahh see my question in the tests - is this what fixes the dynamic validation issues for new test results? I should probably merge in the most recent changes into my social context venue branch...